config/jobs/kubernetes-sigs/sig-windows/release-master-windows.yaml

```diff
@@ -229,7 +229,7 @@ periodics:
   interval: 48h
   decorate: true
   decoration_config:
-    timeout: 2h
```

Comment:
Not a _serial-slow_ job. Please update if we want to revert `ci-kubernetes-e2e-azuredisk-capz-windows-2019` timeout to `2h`